internal/querycoordv2/job/utils.go

```diff
@@ -78,20 +78,15 @@ func loadPartitions(ctx context.Context,
 			return err
 		}
 	}
-	indexes, err := broker.DescribeIndex(ctx, collection)
-	if err != nil {
-		return err
-	}
```

Comment:
we could not simply remove describe index here
index shall exist before loading
any suggestion here @yah01 @weiliu1031